homeassistant/config.py

```diff
@@ -86,19 +85,9 @@
 """
 
 
-def _valid_customize(value):
-    """Config validator for customize."""
-    if not isinstance(value, dict):
-        raise vol.Invalid('Expected dictionary')
-
-    for key, val in value.items():
-        if not valid_entity_id(key):
-            raise vol.Invalid('Invalid entity ID: {}'.format(key))
-
-        if not isinstance(val, dict):
-            raise vol.Invalid('Value of {} is not a dictionary'.format(key))
-
-    return value
+CUSTOMIZE_SCHEMA = vol.Schema({
+    cv.string: vol.All(dict, cv.ordered_dict(cv.string)),
```

Comment:
If you first convert it to a `dict`, then it has no use to convert it to an `ordered_dict`, the order is already lost.